neuralpp/symbolic/sympy_expression.py

```diff
@@ -185,9 +178,32 @@ def sympy_object(self):
     def type_dict(self) -> Dict[sympy.Basic, Expression]:
         return self._type_dict
 
+    @staticmethod
+    def from_sympy_object(sympy_object: sympy.Basic, argument_type: ExpressionType,
+                          type_dict: Dict[sympy.Basic, Expression]) -> SymPyExpression:
+        # Here we just try to find a type of expression for sympy object.
+        if type(sympy_object) == sympy.Symbol:
+            return SymPyVariable(sympy_object, argument_type)
```

Comment:
Any reason to use `type(...) == ...` instead of `isinstance`? ;)

Reply:
no.. I'll change that in my next PR

Reply:
Not really. Gonna change that in my next PR.